easybuild/easyconfigs/m/McXtrace/McXtrace-3.5.24-foss-2023a.eb

```
@@ -0,0 +1,44 @@
+# Author: Erik B Knudsen
+# United Neux
+
+easyblock = 'CMakeMake'
+
+name = 'McXtrace'
+version = '3.5.24'
+
+homepage = 'https://mcxtrace.org'
+description = "McXtrace is a Monte Carlo ray tracing X-ray instrumentaton and experiment simulation packages."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+github_account = 'mccode-dev'
+source_urls = ["https://github.com/mccode-dev/McCode/archive/refs/tags/"]
+sources = ['v%(version)s.tar.gz']
+checksums = ['b1958a24b6b7e06e65ff3fc302da6d306412fdaee15ac77fadf841e1521b773a']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('PyYAML', '6.0')
+]
+
+configopts = '-DBUILD_MCXTRACE=ON -DENSURE_NCRYSTAL=off -DMCVERSION=%(version)s -DBUILD_TOOLS=off -DBUILD_MCXRUN=on'
+
+sanity_check_paths = {
+    'files': ["%(namelower)s/%(version)s/bin/mcxtrace"],
+    'dirs': [f"%(namelower)s/%(version)s/{dd}" for dd in ['samples', 'share', 'optics']]
+}
```

```suggestion
sanity_check_paths = {
    'files': ["bin/mcxtrace"],
    'dirs': ['samples', 'share', 'optics'],
}
```

We really don't want those extra paths. I'd patch their cmakelists to not prepend that extra name + version to their installed material so that it properly respects the --prefix we pas to CMake automatically in CMakeMake easyblock.